### PR TITLE
Misc queue from 2018 (sorry, lost sources)

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -566,23 +566,31 @@ ahve->have
 aicraft->aircraft
 aiffer->differ
 aiport->airport
+air->heir
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: No, on so many levels!

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -566,23 +566,31 @@ ahve->have
...
+airator->aerator
 airbourne->airborne
 aircaft->aircraft
 aircrafts->aircraft
+airial->aerial
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Also suggest arial.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

**Comment by peternewman** [comment]
@larsoner I think this can be merged.

**Pull request merged**